src/main/java/bio/terra/cli/context/GlobalContext.java

```diff
@@ -282,8 +285,22 @@ public CommandRunner getRunner(WorkspaceContext workspaceContext) {
    */
   @JsonIgnore
   public static Path getGlobalContextDir() {
-    // TODO: allow overriding this (e.g. env var != user home directory)
+    // default to the user's home directory
     Path parentDir = Paths.get(System.getProperty("user.home"));
+
+    // if the override environment variable is set and points to a valid directory, then use it
+    // instead
+    String overrideDirName = System.getenv(CONTEXT_DIR_OVERRIDE_NAME);
+    if (overrideDirName != null && !overrideDirName.isBlank()) {
+      Path overrideDir = Paths.get(overrideDirName).toAbsolutePath();
+      if (overrideDir.toFile().exists() && overrideDir.toFile().isDirectory()) {
+        parentDir = overrideDir;
+      } else {
+        throw new SystemException(
```

Comment:
Maybe `IlilegalArgumentException` would be more specific.

Reply:
On second thought, I decided to change this to a `UserActionableException` because it's an error that the user can fix. As opposed to `SystemException` and all other "unexpected" exceptions that are presumed to be a bug in our code. Those three categories correspond to specific CLI exit codes and output formatting, so I think it's good to pick the appropriate category. Thanks for the push to rethink this one.